docs/csharp/language-reference/compiler-options/langversion-compiler-option.md

```diff
@@ -75,11 +76,9 @@ Causes the compiler to accept only syntax that is included in the chosen C# lang
  C# 4.0 [Ecma-334.pdf](https://www.ecma-international.org/publications/files/ECMA-ST/Ecma-334.pdf) Standard ECMA-334 4th Edition    
  C# 5.0 [CSharp Language Specification.docx](https://www.microsoft.com/download/details.aspx?id=7029) C# Language Specification Version 5.0 : Microsoft Corporation  
```

Comment:
This is not related to your PR, but I'm really excited to ask you to update this link. The C# 5 ECMA spec download is here: http://www.ecma-international.org/publications/files/ECMA-ST/Ecma-334.pdf